collect_app/src/main/res/values/strings.xml

```diff
@@ -578,6 +578,18 @@
 
     <string name="islamic_month_11">Dhu al-Qidah</string>
     <string name="islamic_month_12">Dhu al-Hijjah</string>
+    <string name="myanmar_month_1">တန်ခူး</string>
```

Comment:
I think these should be untranslatable just like for the Bikram Sambat calendar. This calendar is only used in Myanmar.

@aurdipas is it ok to use roman numerals? That is, today's date would be displayed as "13 သတင်းကျွတ် 1380" instead of "၁၃ သတင်းကျွတ် ၁၃၈၀"

Reply:
```suggestion
    <string name="myanmar_month_1">တန်ခူး</string>
```

We found that odk collect can also be used in Burmese.
However, we have two types of font recently using in Myanmar.
One is zawgyi and another one is UNICODE standard.
The one in Odk collect is zawgyi  and that means some people cannot read it, because their devices follow unicode standard.
I want to recommend to include both unicode and zawgyi for burmese session.
That is the solution some mobile app developers are using.
We can help you to convert zawgyi text strings to Burmese unicode.

Reply:
Thanks so much for your comment and for making the library, @chanmratekoko! I had no idea about the encoding issue. I just read https://www.globalapptesting.com/blog/zawgyi-vs-unicode and it is fascinating.

Currently, translations are done through Transifex and pulled from there. I wonder whether we can add a Unicode variant there.

Reply:
Also, @chanmratekoko, do you have an opinion on using Roman numerals? Do people in Myanmar tend to know them or not?

Reply:
@lognaturel 
Most of the people from Myanmar know Roman numerals. It is OK.